package.json

```diff
@@ -3,10 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@glennsl/bs-json": "3.0.0",
-    "@rescript/react": "^0.10.3",
-    "bs-fetch": "^0.3.0",
-    "bs-json": "^1.0.1",
+    "@glennsl/rescript-fetch": "^0.2.0",
+    "@rescript/core": "^0.3.1",
```

Comment:
The new official attempt of a standard library in the ReScript language. It comes with all the important stuff you'd expect, including proper `Promise` bindings.